bucket/termius.json

```diff
@@ -21,8 +21,8 @@
         ]
     ],
     "checkver": {
-        "url": "https://autoupdate.termius.com/win/latest.yml",
-        "regex": "version:\\s+([\\d.]+)"
+        "url": "https://docs.termius.com/changelog/desktop",
+        "regex": "<span data-offset-key=\"[^\"]+\">\\s*(\\d+\\.\\d+\\.\\d+) -"
```

Comment:
Test before applying

```suggestion
        "regex": "\">\\s*([\\d.]+)\\s+-\\s+\\d+"
```

Reply:
that regex is wrong

Reply:
> Regex is wrong

Lie.

![image](https://user-images.githubusercontent.com/13260377/134309299-50d6f0b5-9abd-465f-8b16-7361976446b5.png)